tests/model_registry/conftest.py

```diff
@@ -25,39 +25,46 @@
 from pytest_testconfig import config as py_config
 from model_registry.types import RegisteredModel
 from tests.model_registry.constants import (
-    MR_NAMESPACE,
     MR_OPERATOR_NAME,
     MR_INSTANCE_NAME,
     ISTIO_CONFIG_DICT,
     DB_RESOURCES_NAME,
-    MR_DB_IMAGE_DIGEST,
+    MODEL_REGISTRY_DB_SECRET_STR_DATA,
+    MODEL_REGISTRY_DB_SECRET_ANNOTATIONS,
+)
+from tests.model_registry.utils import (
+    get_endpoint_from_mr_service,
+    get_mr_service_by_label,
+    get_model_registry_deployment_template_dict,
+    get_model_registry_db_label_dict,
 )
-from tests.model_registry.utils import get_endpoint_from_mr_service, get_mr_service_by_label
-from utilities.infra import create_ns
 from utilities.constants import Annotations, Protocols, DscComponents
 from model_registry import ModelRegistry as ModelRegistryClient
 
 
 LOGGER = get_logger(name=__name__)
 
-DEFAULT_LABEL_DICT_DB: dict[str, str] = {
-    Annotations.KubernetesIo.NAME: DB_RESOURCES_NAME,
-    Annotations.KubernetesIo.INSTANCE: DB_RESOURCES_NAME,
-    Annotations.KubernetesIo.PART_OF: DB_RESOURCES_NAME,
-}
+
+@pytest.fixture(scope="class")
+def dsc_resource_scope_class(admin_client: DynamicClient) -> DataScienceCluster:
+    return DataScienceCluster(name=py_config["dsc_name"], ensure_exists=True)
+
+
+@pytest.fixture(scope="class")
+def model_registry_namespace_name(dsc_resource_scope_class: DataScienceCluster) -> str | None:
```

Comment:
fixture does not return str
also please fix the name of the fixture

Reply:
What do you mean? dsc_resource_scope_class.instance.spec.components.modelregistry.registriesNamespace is a string.
```
dbasunag@dbasunag-mac opendatahub-tests % oc explain dsc.spec.components.modelregistry.registriesNamespace
GROUP:      datasciencecluster.opendatahub.io
KIND:       DataScienceCluster
VERSION:    v1

FIELD: registriesNamespace <string>


DESCRIPTION:
    Namespace for model registries to be installed, configurable only once when
    model registry is enabled, defaults to "rhoai-model-registries"
    

dbasunag@dbasunag-mac opendatahub-tests % 
```